packages/storybook-vue/src/stories/BadgeStatus/BadgeStatus.stories.mdx

```diff
@@ -0,0 +1,180 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { BadgeStatus } from "@utrecht/component-library-vue";
+import { argTypes } from "./argTypes.ts";
+
+<Meta id="vue-badge-status" title="Vue.js Component/BadgeStatus" component={BadgeStatus} argTypes={argTypes} />
+
+# BadgeStatus
+
+export const Template = (args) => ({
+  components: { BadgeStatus },
+  setup() {
+    return { args };
+  },
+  template: `<BadgeStatus v-bind="args">${args.default}</BadgeStatus>`,
+});
+
+<Canvas>
+  <Story
+    name="Badge Status Neutral"
+    args={{
+      status: argTypes.status.options[0],
+      default: argTypes.status.options[0],
```

Comment:
Seems very tricky to refer to `options[n]`, reordering the array would lead to stories that don't match their titles. Is there a significant disadvantage to simply hardcoding the values here?

Reply:
I find it easier to add, and update, but we can use hard coded instead, WDYT?

Reply:
I would prefer hardcoded as the person updating the options in args will not necessarily know they'll cause side-effects elsewhere. 

I get it's easier to update now, but it causes mental overhead too to keep track of positional references like this